src/Fantasma.Sandbox/DatabaseContext.cs

```diff
@@ -0,0 +1,16 @@
+using Fantasma.EntityFramework;
+using Microsoft.EntityFrameworkCore;
+
+namespace Fantasma.Sandbox;
+
+public class DatabaseContext : DbContext, IFantasmaDatabase
+{
+    public DbSet<FantasmaJob> Jobs { get; set; }
```

Comment:
```suggestion
    public DbSet<FantasmaJob> Jobs => Set<FantasmaJob>();
```

Should help with nullable warnings